### PR TITLE
Add Chang Chain Foundation Mainnet

### PR DESCRIPTION
**Pull request by earthchie**
This PR adds the Chang Chain Foundation Mainnet to the Chainlist.

- Name: Chang Coin Foundation Chain
- RPC: https://rpc.cthscan.com/
- Chain ID: 5858
- Currency symbol: CTH
- Explorer: https://cthscan.com/

Please let me know if any changes are needed. Thank you!

### CODE REVIEW

File: _data/icons/changchain.json [hunk 1]
```diff
@@ -0,0 +1,9 @@
+[
+    {
+      "url": "ipfs://QmbEytMCh2xFr7q87HjthX4K2MKmjfHih2mqS42tqTZoMy",
```
Comment (sreekv143) [inline _data/icons/changchain.json:3]: Ad1489

Comment (sreekv143) [inline _data/icons/changchain.json:3]: Help me

**Review by sreekv143 — COMMENTED** [review]
Sooper

**Review by sreekv143 — COMMENTED** [review]
sreeleshkv1489@gmail.com

File: _data/icons/changchain.json (hunk 1 above)
Comment (Mosan12) [reply _data/icons/changchain.json:3]: @mosan12

Comment (Mosan12) [reply _data/icons/changchain.json:3]: @Mosan12

**Comment by earthchie** [comment]
I'm not sure why the IPFS check is failing with the error "Error: Process completed with exit code 1" again. I’ve tested the file with multiple IPFS gateways, and they all seem to be working fine:

https://gateway.ipfs.io/ipfs/QmbEytMCh2xFr7q87HjthX4K2MKmjfHih2mqS42tqTZoMy

https://cloudflare-ipfs.com/ipfs/QmbEytMCh2xFr7q87HjthX4K2MKmjfHih2mqS42tqTZoMy

https://bafybeif7wqqnhwsiyqjbm6xpoanfeo3cm6lfizwowiiugzoxdw4sui2oda.ipfs.dweb.link

https://bafybeif7wqqnhwsiyqjbm6xpoanfeo3cm6lfizwowiiugzoxdw4sui2oda.ipfs.dweb.link

https://bafybeif7wqqnhwsiyqjbm6xpoanfeo3cm6lfizwowiiugzoxdw4sui2oda.ipfs.4everland.io

https://gateway.pinata.cloud/ipfs/bafybeif7wqqnhwsiyqjbm6xpoanfeo3cm6lfizwowiiugzoxdw4sui2oda

https://bafybeif7wqqnhwsiyqjbm6xpoanfeo3cm6lfizwowiiugzoxdw4sui2oda.ipfs.dweb.link

If I’ve misunderstood the issue or if there’s anything else I should address, please let me know and I’ll make the necessary adjustments.

**Comment by earthchie** [comment]
Hi @ligi. This PR has been stuck here for a while, could you review it, please?

**Comment by earthchie** [comment]
Hi @ligi, sorry to bother you again. After checking the workflow script I realized that I could fix this gateway problem by just putting the icon in the `iconsDownload` folder. I should have realized this sooner. sorry :(

**Review by ligi — APPROVED** [review]

**Comment by earthchie** [comment]
Wow thanks!

**Pull request merged**